tests/classifiersFrameworks/conftest.py

```diff
@@ -0,0 +1,2 @@
+import pytest
+import numpy as np
```

Comment:
Where are these imports used?

Reply:
they were part of fixtures that were specific to classifiers but where removed after some more refactoring. technically this conftest file should be completely removed until we have new classifier specific fixtures that arise again. The only reason I left it there for now was as a reminder for ourselves that pytest can use package specific fixtures but we now have one as well for attacks so I guess it's no longer necessary. is creating a new commit with this file removed the best way of doing this? I guess that means we'll have to wait for all the tests to pass again?